src/NServiceBus.AcceptanceTests/ApprovalFiles/When_endpoint_is_warmed_up.Make_sure_things_are_in_DI.approved.txt

```diff
@@ -1,5 +1,7 @@
 ----------- Used registrations (Find ways to stop accessing them)-----------
 NServiceBus.InferredMessageTypeEnricherBehavior - Transient
+NServiceBus.Persistence.ISynchronizedStorage - Singleton
```

Comment:
> Or is it just that since we now have an outbox capable storage for testing they show up since we are able to enable the outbox?

Yeah, that's the reason.